README.md

```diff
@@ -72,4 +72,29 @@ Persistent settings
 
 ### SSH to the Pi
 
-1. In your terminal/powershell type `ssh pi@IPADDRESS` to log into the Pi server with the username `pi` and the password `raspberry`.
\ No newline at end of file
+1. In your terminal/powershell type `ssh pi@IPADDRESS` to log into the Pi server with the username `pi` and the password `raspberry`.
+
+### SSH Keys
+
+1. Create SSH key pair from workstation to server
+    1. Type `ssh-keygen` into your workstation terminal
+    2. Press Enter to accept the default directory `~/.ssh/id_rsa`
+    3. If the default directory already exists you will be asked to Overwrite. Press `y` then Enter
+
+    * Keep in mind this will overwrite your current key pair. Any existing devices that are key paired will no longer authenticate using the previous key.
+    4. Press Enter to accept no passphrase
+    5. Press Enter again to confirm no passphrase
+
+    * Your public key is now saved at `~/.ssh/id_rsa.pub`
+
+2. Copy your public SSH key to your server
+    1. Linux/MacOS: `ssh-copy-id -i ~/.ssh/id_rsa.pub <USERNAME>@<SERVER-IP>`
+    2. Windows: `cat ~/.ssh/id_rsa.pub | ssh <USERNAME>@<SERVER-IP> "cat >> ~/.ssh/authorized_keys"`
```

Comment:
This step fails from **windows** to **raspios** due to directory & file `~/.ssh/authorized_keys` non existing on the pi.

... manually creating the path on the pi fixes the issue